### PR TITLE
Task3

### PR DESCRIPTION
**Pull request by Kernstall**
Task 3 ready for a review @djadjka

### CODE REVIEW

File: task3/styles/styles.css [hunk 1]
```diff
@@ -0,0 +1,313 @@
+body{
+    margin: 0px;
```
Comment (tyndria) [inline task3/styles/styles.css]: you could omit px when value is zero

File: task3/styles/styles.css [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+.logOutH3{
+
+    font-size: 20px;
+    font-family: sans-serif;
+    text-align: center;
+    color: #fefac0;
+    margin: 0px;
+    padding-top: 14px;
+}
+a{
```
Comment (tyndria) [inline task3/styles/styles.css]: it's bad practice to redefine tag's styles. please, define class with properties and use this class on tag 'a'

File: task3/styles/styles.css [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+    background-color: black;
+}
+.postBodyHolder{
+    height: 100%;
+}
+.hashTag{
+    height: 20px;
+    font-size: 16px;
+    background-color: cornflowerblue;
+    border-radius: 8px 8px 8px 8px;
```
Comment (tyndria) [inline task3/styles/styles.css]: read about shorthand properties 
https://www.w3schools.com/cssref/css3_pr_border-radius.asp

File: task3/index.html [hunk 4]
```diff
@@ -0,0 +1,418 @@
...
+        <div class="feedList">
+            <div class="postWrapper">
+                <div class="posterInfo">
+                    <h5 class="posterName"> Alex B. </h5>
+                    <p class="postTime">20:15</p>
+                </div>
+                <div class="postBodyHolder">
+                    <div class="picAndDescrHolder">
+                        <img class="postImg" src="resources/Adam.jpg">
+                        <figure class="postDescription">
```
Comment (tyndria) [inline task3/index.html]: wrong usage of semantic figure tag, please, see https://www.w3schools.com/tags/tag_figure.asp

File: task3/index.html [hunk 5]
```diff
@@ -0,0 +1,418 @@
...
+                </div>
+            </div>
+
+            <div class="loadMore">Load More</div>
+        </div>
+
+        <div class="filterPanel">
+            <div class="stickyFilterScroller">
+                <div class="filter">
+                    <p style="margin-left: 10px; font-size: 18px;">Filter properties:</p>
```
Comment (tyndria) [inline task3/index.html]: inline styles is not a good practice. it's better to define it in .css file using class

**Comment by tyndria** [comment]
please, try to use semantic HTML tags, like \<main\>, \<section\>, \<article\> and etc

**Comment by Kernstall** [comment]
Should i use <main> for my whole feed list? I read is should be unique

**Pull request merged**